### PR TITLE
GraphQL Sydney December 2023

### PR DESCRIPTION
**Pull request by danstarns**
Upcoming meetup in Sydney. 

CC @joshprice @hasura

### CODE REVIEW

File: events.json [hunk 1]
```diff
@@ -1,4 +1,14 @@
 [
+  {
+    "name": "Sydney December",
+    "slug": "graphql-sydney-01",
+    "location": "Sydney",
+    "date": "2023-12-05T18:00:00+11:00",
+    "coverImage": "TODO",
```
Comment (mbuhot) [inline events.json:7]: GraphQL December 2023

File: events.json [hunk 2]
```diff
@@ -1,4 +1,14 @@
...
+    "eventLink": "TODO",
```
Comment (mbuhot) [inline events.json:8]: https://www.meetup.com/graphql-sydney/events/297415925/

File: events.json [hunk 3]
```diff
@@ -1,4 +1,14 @@
...
+    "host": "Microsoft",
```
Comment (mbuhot) [inline events.json:9]: Microsoft Reactor

File: events.json [hunk 4]
```diff
@@ -1,4 +1,14 @@
...
+    "hostLink": "https://www.microsoft.com/"
```
Comment (mbuhot) [inline events.json:10]: https://developer.microsoft.com/en-us/reactor/

File: events.json (hunk 2 above)
Comment (mbuhot) [reply events.json]: or alternatively: https://graphql.sydney

**Review by mbuhot — APPROVED** [review]
👍 Thanks @danstarns

**Comment by danstarns** [comment]
Screenshot 2023-12-05 at 15 44 20

Live, thanks @mbuhot

**Pull request merged**